portfolio.html

```diff
@@ -0,0 +1,105 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <title>Portfolio page</title>
+        <meta name="viewport contents" content="device-width, initial-scale=1.0">
+        <link rel ="stylesheet" type="text/css" href="portfolio-style.css">
+        <link rel ="stylesheet" type="text/css" href="normalise.css">
+    </head>
+    <body>
+        <header>
+
+            <section class="top-bar">
+                <div class="name">
+                    <h3>Matthew Hammond</h3>
+                </div>
+                <div class="top-links">
+                    <a href="#bio"><button type="button">Bio</button></a>
+                    <a href="#portfolio"><button type="button">Portfolio</button></a>
+                    <a href="#contact"><button type="button">Contact me</button></a>
+                </div>
+            </section>
+
+            <section class="landing">
+                <div class="landing-container">
+                    <img class="landing-image" src="./Assets/aerial.jpg">
```

Comment:
img tags need an alt to be compliant with HTML standards, this helps users who are using screen readers.

^ Applies to all img tags